frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/vcsmenu/OpenMacheteTabAction.java

```diff
@@ -0,0 +1,40 @@
+package com.virtuslab.gitmachete.frontend.actions.vcsmenu;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.project.DumbAwareAction;
+import com.intellij.openapi.wm.ToolWindow;
+import com.intellij.openapi.wm.ToolWindowId;
+import com.intellij.openapi.wm.ToolWindowManager;
+import lombok.CustomLog;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
+
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeyProject;
+import com.virtuslab.logger.IEnhancedLambdaLogger;
+
+@CustomLog
+public class OpenMacheteTabAction extends DumbAwareAction implements IExpectsKeyProject {
+  @Override
+  public IEnhancedLambdaLogger log() {
+    return LOG;
+  }
+
+  @Override
+  @UIEffect
+  public void actionPerformed(AnActionEvent e) {
+    LOG.debug("Performing");
+
+    ToolWindowManager toolWindowManager = ToolWindowManager.getInstance(getProject(e));
+    ToolWindow toolWindow = toolWindowManager.getToolWindow(ToolWindowId.VCS);
+    if (toolWindow == null) {
+      LOG.debug("VCS tool window does not exist");
```

Comment:
What about vcs notification? This one is probably gonna happen when git is not used within project. So maybe:
`Could not open Git Machete tab. Ensure that Git VCS is enabled.`?

Reply:
It sounds reasonable, so I done this. This is how it's working:
![Screenshot from 2020-07-02 15-52-51](https://user-images.githubusercontent.com/56402877/86367553-62562200-bc7c-11ea-96b5-9a6de6081c4f.png)
In addition - `toolWindow` seems to be **not** null even when there is no CVS repository available in opened project but then `tab` is `null`, so I added additional check.